Nicholas Long
============

----

>  Computational Biologist\
> <noplacelikestardust@gmail.com> • (214) 236 4183\
> https://github.com/n-long

----

Education
---------

2013-2016
:   **M.S., Biology**; University of Texas at Arlington

    *Thesis title: Gene Expression Misregulation in Hybrids With\
     Neuromuscular Disorder*

2009-2013
:   **B.S., Exercise Science (Clinical Professions Track)**; UT Arlington

    *Minor: Biology, Chemistry*

Research
----------

* Rapidly assimilate literature and analytical techniques to answer self-directed research questions

* Compare algorithms for efficient automation across large volumes of sequencing data

* In-depth knowledge of statistical models and trade-offs in specific use cases

* Query databases for meaningful intersection of results across multiple analyses 

Technical Experience
--------------------

Unix
:   Linux server maintenance

    * Evaluate, install, and troubleshoot bioinformatics software
    * Extensive bash scripting for data formatting and analysis pipelines
    * Distribute resources with Sun Grid Engine job scheduler

R
:   Detected gene expression changes across timepoints for a neuromuscular disease

    * Gene co-expression networks for biological function of large mis-expressed clusters
    * Multi-factorial design for contrasts of gene expression changes across a time-course
    * Create publication-ready graphics 

Python
:   Developed pipeline based on mitochondrial sequencing reads for genome assembly and gene annotation

    * Searches custom-built gene profiles
    * Filters by alignment score and length
    * Outputs sequence and coordinates in common format

Teaching
---------
**Graduate Teaching Assistant**

* Zoology Lab: Fall 2013 - May 2015

* Anatomy & Physiology Lab: Summer 2015 - Spring 2016

**Software Carpentry Workshop Instructor (1/20/16)**

* Unix lesson covering loops and bash scripts to audience of 40
* Assisted other sections in Python, R, and Git

**Bioinformatics tutoring**

* Provided project and homework guidance for Python-based bioinformatics course\
  (mixed undergraduate / graduate)

Graduate Courses 
----------------------------------------
* Bioinformatics

* Molecular Evolution

* Population Genetics

* Biological Statistics

* Biological Modeling

* Genome Structure and Dynamics

* Advanced Molecular Biology

* Genetics Methods Lab

Publications
---------------------------------------
Watson, E.T., N.A. Long, P.D. Glenn, and J.P. Demuth. 2016. Resistance is infertile: Mitonuclear compatibility in Tribolium castaneum is associated with history of insecticide resistance.\
(In preparation)  

References
----------------------------------------  
Jeffery Demuth  
Associate Professor (Advisor)  
(817) 272-2653 &nbsp;&nbsp;• &nbsp;&nbsp;jpdemuth@uta.edu

Matthew Fujita  
Assistant Professor  
(817) 272-1464 &nbsp;&nbsp;• &nbsp;&nbsp;mkfujita@uta.edu  

Eric Watson  
Postdoctoral scholar  
(817) 897-0756 &nbsp;&nbsp;• &nbsp;&nbsp;ericwats@usc.edu

----

> <nlong@uta.edu> • (214) 236 4183\
> https://github.com/n-long
